app/assets/stylesheets/custom.scss

```diff
@@ -114,3 +114,17 @@ body {
 .price {
   font-size: 1.5em;
 }
+
+// dashboard stuff
+
+.dad-invoice-heading h3 {
+  padding-top: 20px;
```

Comment:
Properties should be ordered border-bottom, border-top, padding-bottom, padding-top